src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateImplicitNamespaceImports_DotNet.cs

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToGenerateImplicitNamespaceImports_DotNet : SdkTest
+    {
+        public GivenThatWeWantToGenerateImplicitNamespaceImports_DotNet(ITestOutputHelper log) : base(log) { }
+
+        [CoreMSBuildOnlyFact]
```

Comment:
Restricting some of these tests to Core until I figure out what the C# 10 support story is on Full Framework.

Reply:
The problem with full framework is probably that the version of Visual Studio installed on the CI machines, and which is used for the Full Framework tests, is not recent enough.

Instead of using `CoreMSBuildOnlyFact`, try `[RequiresMSBuildVersionFact("17.0.0.32901")]`.  That will make the tests automatically start running on Full Framework once it is updated to a recent enough version.

Reply:
Ah that's awesome